currency/currency_service.bal

```diff
@@ -76,8 +76,9 @@ isolated function parseCurrencyJson(json currencyJson) returns map<decimal>|erro
     map<decimal> currencies = {};
     map<string> currencyValues = check currencyJson.cloneWithType();
 
-    foreach string key in currencyValues.keys() {
-        currencies[key] = check decimal:fromString(currencyValues.get(key));
-    }
+    check from string key in currencyValues.keys()
+        do {
+            currencies[key] = check decimal:fromString(currencyValues.get(key));
+        };
```

Comment:
You can do this in a single query expression.

```bal
map<decimal> currencies = map from string key in currencyValues.keys()
        select [key, check decimal:fromString(currencyValues.get(key))];
```

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/30/commits/2e2d24dc6ca367a370b837260a006b7265ceef8c